### ISSUE TITLE
Somewhere between the glm and stan

### ISSUE THREAD

**Issue by klahrich** [tidymodels/embed#4]
Would it be interesting to have an option to let the user choose how to blend the prior and posterior estimates through a hyperparameter (versus lme4 or stan) ?

Here's a description of the approach: https://kaggle2.blob.core.windows.net/forum-message-attachments/225952/7441/high%20cardinality%20categoricals.pdf.

I've seen it used on kaggle a lot. Usually users also add a random noise parameter as another hyperparameter to be added to the coefficient estimates.

It should be easy to implement, would happy to do it. Could be on top of the glm approach (as a last optional step after the glm coefficients have been estimated) or a separate method.

**Comment by artichaud1** [comment]
Any thoughts on this?
I realize my text was a bit too long...
TLDR: control the blending of prior and posterior with a hyperparam. Simpler than bayesian, easy to tune.

**Comment by topepo** [comment]
Go ahead and do a PR if it's easy. I don't particularly like it since you could get a lot of the same effect by changing the prior for the intercept (to be more narrow) or my using the `glmnet` etc. 

Then again, I haven't tried it yet so ¯\_(ツ)_/¯

Either way, it's good to have another (quasi) reference.

**Comment by topepo** [comment]
Closed due to inactivity.
